packages/patrol_cli/lib/src/runner/patrol_command.dart

```diff
@@ -117,6 +117,14 @@ abstract class PatrolCommand extends Command<int> {
     );
   }
 
+  void usesMacOSOptions() {
+    argParser.addOption(
+      'bundle-id',
+      help: 'Bundle identifier of the MacOS app under test.',
+      valueHelp: 'pl.leancode.macos.AwesomeApp',
+    );
+  }
+
```

Comment:
Doesn't it conflict with `usesIosOption()`? Some assertion should be done here to prohibit incorrect use.

Reply:
`ArgParser` has built-in checking for duplicated commands.

Reply:
ah, alrighty then